src/components/organisms/MobileNav/index.tsx

```diff
@@ -1,28 +1,32 @@
 import React, { useState } from 'react'
+import data from "../../../../config/translations/en.json"
```

Comment:
`data` should come from props. What if the language is Japanese?

Reply:
I don't know. I can't figure out how to do that with imports.

Reply:
Check my previous PR 😄 
https://github.com/Alfrex92/animers/pull/4

Reply:
Okay, I think I got it this time lol